TACC/Core-Styles
# enhance: white headings in s-footer

## Overview / Changes

- Headings in an `s-footer` become white.
- New prototype demo for complex `s-footer`.

## Related

- required by https://github.com/TACC/tup-ui/pull/476

## Testing

1. Open http://localhost:3000/components/detail/s-footer--complex.
2. Verify headings are white.
3. Verify text is light gray.

## UI

<img width="920" alt="Core-Styles #391" src="https://github.com/user-attachments/assets/cf5a1d2c-b125-43e7-8558-399a8f9f96be">
